pkg/grapicmd/internal/usecase/initialize_project_usecase.go

```diff
@@ -10,7 +10,9 @@ import (
 
 	"github.com/izumin5210/grapi/pkg/cli"
 	"github.com/izumin5210/grapi/pkg/excmd"
-	"github.com/izumin5210/grapi/pkg/grapicmd/internal/module"
+	"github.com/izumin5210/grapi/pkg/gencmd"
+	_ "github.com/izumin5210/grapi/pkg/grapicmd/internal/usecase/template"
```

Comment:
**[golangci]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
a blank import should be only in a main or test package, or have a comment justifying it (golint)